proto/metapb.proto

```diff
@@ -61,6 +61,16 @@ message RegionEpoch {
     uint64 version = 2;
 }
 
+message Buckets {
+    uint64 region_id = 1;
+    uint64 version = 2; // A hint indicate if keys have changed.
+    repeated bytes keys = 3;
+    repeated uint64 read = 4;
+    repeated uint64 scan = 5;
+    repeated uint64 write = 6;
```

Comment:
How about using [QueryStats](https://github.com/pingcap/kvproto/blob/master/proto/pdpb.proto#L772) here?

Reply:
Today pdpb.proto depends on this file.  This file cannot import pdpb.proto or it will leads to cycle import.

Reply:
But I think it's a good idea to wrap all stats data into a separate BucketStats message, which will be easier to extend in the future.